### PR TITLE
Update executionModes.md

### PR DESCRIPTION
**Pull request by Geheiner**

### CODE REVIEW

File: docs/reference/executionModes.md [hunk 1]
```diff
@@ -258,22 +263,7 @@ This can be achieved by using FileIncrementalMoveMode. If option `archiveSubdire
 
 FileIncrementalMoveMode can be used with the file engine (see also [Execution engines](executionEngines.md)), but also with SparkFileDataObjects and the data frame engine.
 
-## Others
-
-### ProcessAllMode
-An execution mode which forces processing of all data from its inputs.
-Any partitionValues and filter conditions received from previous actions are ignored.
-
-### CustomPartitionMode
```
Comment (Geheiner) [inline docs/reference/executionModes.md:267]: this is already mentioned above, thus I removed the duplicated paragraphs

File: docs/reference/executionModes.md [hunk 2]
```diff
@@ -258,22 +263,7 @@ This can be achieved by using FileIncrementalMoveMode. If option `archiveSubdire
...
-This execution mode allows to implement arbitrary custom partition logic using Scala.
-
-Implement trait `CustomPartitionMode` by defining a function which receives the main input and output DataObjects
-and returns the partition values that need to be processed.
 
-### CustomMode
```
Comment (zzeekk) [inline docs/reference/executionModes.md:273]: CustomMode is not mentioned above. We should keep this text.

File: docs/reference/executionModes.md [hunk 3]
```diff
@@ -183,8 +183,13 @@ If you define `nbOfPartitionValuesPerRun`, PartitionDiffMode will only process t
 
 ### CustomPartitionMode
 This execution mode allows for complete customized logic to select partitions to process in Scala.
-Implement trait `CustomPartitionModeLogic` by defining a function which receives main input & output DataObjects and returns partition values to process as `Seq[Map[String,String]]`
+Implement trait `CustomPartitionModeLogic` by defining a function which receives main input & output DataObjects and returns partition values to process as `Seq[Map[String,String]]`.
+The contents of the command-line parameters `--partition-values` and `--multi-partition-values` is provided in the input argument `givenPartitionValues`.
+You are free to use or ignore this information in your custom execution mode. You can also use a `CustomPartitionMode` together with the Default Behavior in the same DAG run by having some Actions define a `CustomPartitionMode` and others not. For example, you can partition measurement data by day and select individual days using `--partition-values`, but fetch master data based on a different, custom logic.
```
Comment (zzeekk) [inline docs/reference/executionModes.md]:
```suggestion
You are free to use or ignore this information in your custom execution mode. You can also use a `CustomPartitionMode` together with the Default Behavior in the same DAG run by having some Actions define a `CustomPartitionMode` and others not. For example, you can partition transaction data by day and select individual days using `--partition-values`, but fetch master data based on a different, custom logic.
```

**Review by zzeekk — APPROVED** [review]
Ok for me when all comments are handled

File: docs/reference/executionModes.md (hunk 2 above)
Comment (Geheiner) [reply docs/reference/executionModes.md:273]: done

**Pull request merged**